src/components/Button.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name, wide, color } = props;
+  const {
+    name, wide, color, clickHandler,
+  } = props;
 
   return (
-    <button className={`button ${wide ? 'button-double' : ''}`} type="button" style={{ background: color }}>
+    <button className={`button ${wide ? 'button-double' : ''}`} type="button" style={{ background: color }} onClick={() => { clickHandler(name); }}>
       {name}
     </button>
```

Comment:
**Button component**
- It implements `handleClick` function  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png) then `handleClick` should be passed as `clickHandler` props to each `button` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/button/callClickOnButton.png)